<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>183</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>本地IP:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>本地端口：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_port"/>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>50</y>
     <width>21</width>
     <height>361</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>201</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>节点一：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_state">
        <property name="text">
         <string>断线</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>电流：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_current"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>MA</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>电压：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_volot"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>V</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>累计电能：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_kwh"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>KWH</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>201</width>
     <height>104</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>节点二：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_state_2">
        <property name="text">
         <string>断线</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>电流：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_current_2"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>MA</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>电压：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_volot_2"/>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>V</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>累计电能：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_kwh_2"/>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>KWH</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>201</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>节点三：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_state_3">
        <property name="text">
         <string>断线</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>电流：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_current_3"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>MA</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>电压：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_volot_3"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>V</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_21">
          <item>
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>累计电能：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_kwh_3"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>KWH</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>201</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_22">
      <item>
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>节点四：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_state_4">
        <property name="text">
         <string>断线</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_23">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_24">
          <item>
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>电流：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_current_4"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>MA</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_25">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_26">
          <item>
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>电压：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_volot_4"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>V</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_27">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_28">
          <item>
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>累计电能：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_kwh_4"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>KWH</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>411</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>201</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_29">
      <item>
       <widget class="QLabel" name="label_31">
        <property name="text">
         <string>节点五：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_state_5">
        <property name="text">
         <string>断线</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_30">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_31">
          <item>
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>电流：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_current_5"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>MA</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_32">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_33">
          <item>
           <widget class="QLabel" name="label_34">
            <property name="text">
             <string>电压：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_volot_5"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_35">
          <property name="text">
           <string>V</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_34">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_35">
          <item>
           <widget class="QLabel" name="label_36">
            <property name="text">
             <string>累计电能：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_kwh_5"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_37">
          <property name="text">
           <string>KWH</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>201</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_36">
      <item>
       <widget class="QLabel" name="label_38">
        <property name="text">
         <string>节点六：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_state_6">
        <property name="text">
         <string>断线</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_37">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_38">
          <item>
           <widget class="QLabel" name="label_39">
            <property name="text">
             <string>电流：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_current_6"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_40">
          <property name="text">
           <string>MA</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_39">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_40">
          <item>
           <widget class="QLabel" name="label_41">
            <property name="text">
             <string>电压：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_volot_6"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_42">
          <property name="text">
           <string>V</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_41">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_42">
          <item>
           <widget class="QLabel" name="label_43">
            <property name="text">
             <string>累计电能：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_kwh_6"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_44">
          <property name="text">
           <string>KWH</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_45">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/logo.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_46">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>BY--建智161（专升本）--周跃琪</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
